Sources/Views/SubscriptionConfirmation.swift

```diff
@@ -111,6 +126,11 @@ private func teamMembers(_ currentUser: User, _ subscribeData: SubscribeConfirma
                   [("id", "team-member-template")],
                   [teamMemberTemplate("", withRemoveButton: true)]
                 ),
+                .element(
+                  "template",
+                  [("id", "team-member-template-without-remove")],
+                  [teamMemberTemplate("", withRemoveButton: false)]
+                ),
```

Comment:
I need a template of the team member row without a remove button because that is what will replace the owner if they decide to remove themselves.